third_party/nvidia/backend/compiler.py

```diff
@@ -107,6 +107,8 @@ class CUDAOptions:
     cluster_dims: tuple = (1, 1, 1)
     ptx_version: int = None
     enable_fp_fusion: bool = True
+    global_scratch_align: bool = True
+    global_scratch_size: int = 0
```

Comment:
what does the default value means? This field has to match what the kernel decides, it feels meaningless to have a default values